src/activitylog/activitylog.html

```diff
@@ -1 +1,46 @@
-<!-- activitylog.html -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Extension Activity Monitor</title>
+    <link rel="stylesheet" href="activitylog.css" />
+    <script src="activitylog.js" type="module"></script>
+  </head>
+  <body>
+    <div class="container">
+      <h1>Extension Activity Monitor</h1>
+      <div class="load-file-wrapper">
+        <input id="logInputFile" type="file" accept="application/json" />
+        <button id="loadLogBtn" disabled>Load logs</button>
+      </div>
+      <button id="saveLogBtn">Save logs</button>
+      <div class="notice"></div>
+      <div class="activity-log-wrapper">
+        <h2>Activity Logs</h2>
+        <div class="activity-log-container">
+          <div class="log-table">
+            <table>
+              <tr>
+                <th>Extension Id</th>
+                <th>Timestamp</th>
+                <th>API Type</th>
+                <th>API Name</th>
+                <th>View Type</th>
+              </tr>
```

Comment:
Nit, this `<tr>` should be moved into a `<thead>` tag otherwise the table is going to have two `<tbody>` child elements (because the browser creates one automatically to move this `<tr>` tag into it, and then the row created using `insertRow` will be added to the `<tbody>` created automatically, and the `<tbody>` that follows at line 31 will still be there in the DOM but empty).

Reply:
done :-) Sorry I missed this out earlier.